models/user.go

```diff
@@ -1,8 +1,19 @@
 package models
 
+import "time"
+
 type User struct {
 	ID       uint   `gorm:"primary_key"`
 	Username string `gorm:"not null;unique" json:"username" binding:"required"`
 	Password string `gorm:"not null" json:"password" binding:"required"`
 	Email    string `gorm:"not null;unique" json:"email" binding:"required"`
 }
+
```

Comment:
new struct type for LoginVerification of the user if he logs in from another device which IP address is not registered in the database